docs/source/concept_models/vivarium_ciff_sam/DAG_acute_malnutrition.drawio

```diff
@@ -0,0 +1 @@
WyqaBwfWnajtv08ur89Pz1cXuXEtmPLBGLLB5zLz+Qe9nAvSNjq9mqGXzglmNGFzHYQxe1a34HXRmRLXko1eIDbFvW+tk/apUeqUzlqfayp3Ntpx6GhS0qbfL9bQVeVcpOdWXyqWu1zF085sXSmnJWjzsvqpm25et2MAdRM63m5ePmCoT9SIQwJrQyFonyQGwRQ6Fd+m1zVkufSW1Ri5+b85GvYby/fXAniIRGwAUxcMGBwJWRUNzFMB+f+vOHK6WZKkuT9qFzgNj1t27he7fw1s8ebErrULozvqULrVTfvhA3MXn7CUGPe+ekobc6zqARX7Q2qzNVmav+6SrmnzDC58LU2hbIQEBflAnaUdtm3uGt7mlotyPa7wjznY4aYAYd1XTZTPvhDGrPt4+i0LWpPINAIhkP5e912QdNU97yyBfWncJnbY2bhbZ7pxZ6kdSiUCacS0CBW8O4I0SfaBIlBAnG3R4aYgofSjsmEyQI4hARTtjOskEQitU+FSWHyYAHUxImwAPdoSR5RkLiOxISHRUhfx+tI48n1v7IwF6kOv0+ZBhmAB6uMh2FgTfqy/7ow+9IH3ZU9MGEJFIA4NY3yAmigvZJgGkCASauTlKA0c7Cc2duAZZIjBnjGEMBaT/7hDlOAGYQIMkx2aQFn4p3w9TE/XnX7tmBW95od98OEnbLzr0uHW6JEy+SlsjRUkggOJKMItbeDbVHPcJjtfdlJRIo4ZByJNzmjWLR1z8kaKx2/6ycp5ff9Dfz9d/1p1Oxujw9N2QSuX9p8x/lQuWPSrf0Utp1AnuuMmWN8t/kbQurp7nK03Un5dKM+hJSLru5bRHJztEhcVvbY1ICUU+i/S1rPsFx0IXtJiq+NAVB3C0qHENJ3YmJsZ1LPDdMtPreGsJFAiTBWDKEGN5NSaEEuq4hxAg4FWGHkhAm8o8TcaYJuTyzKzNNLxc2IiF/795clU4J7D2uPi6e4upjEOoR2v766kPSHReQQfgf7uoj6od9CEA0Jxjk4B5ZwY2Q9E7g9EYhiUb1rUpyEyPvdsJ6tnQPZglwe1QgpItn2HqrPV4xmGDEOebMqYcAD5QNImTCmaROExDkAFDSHwWKaPWjlcMVt38saZpv/56D3/0P</diagram><diagram id="RaqAcknEyuArPcIOtZxW" name="Page-2">3Vpdc+MmFP01nmkf4uFDIOkxziZtppvdTDPT3e0bK2GbRgYHodjeX79ggW19pHGa2EmaBwUuAumeezhckAf4bLb8TbP59ErlvBggkC8H+MMAIYghsv+cZVVbEuANEy3y2gS2hhvxg/uewVqJnJfeVpuMUoUR86YxU1LyzDRsTGu1aN42VkXeMMzZhHcMNxkrutYvIjdT7wUBW/vvXEym4ckQ+JYZCzd7QzlluVrsmPD5AJ9ppUxdmi3PeOHAC7jov+4ubmeLC5WSH+UZP/38x+f8pB7s4ildNi5oLs1/HvpqReTXxcWI/n15Kk00Gcm70ncB96yoPF6nWWX4wDlIC/uw0XdtSxNXmrFCVkYLI5T0iJhVgNk+y0bUVkaLqTD8Zs4y17KwpLK2qZkVtgZtkZXzOsxjseT21Ub+Dbg2fNkK2SP+wk0QLHu5mnGjV7afHyWKknocT1xIo7q+2NIAAm+b7lIgGJmn3mQz9hZeW/AIPwFt2EH7Uo4tGkJVpZtiouSsdODzybpq+T/lzL2JzO3145+X7wB3FHD2uOMo7sGd9uFOD4Q76uD+hZVGSIeyFuWt0xWWGaWd95rfW/ffB8chaHI8SkAX66DE06bMHQhr3MH6UxCNEweLGIusBTMQ0nDtq3YiUDZzgMrv5XyD3dsOA22GISH0tcNAOmEYCW2mi/rh7w9iROKWmndVJe0R8/RQWk47AIcVc6ysn7tw0rtKhYaTcp0hndobIJwvt41hlR2xgsmMO73nkuvJaj1ZlOFC9q7KvximJ9yLWSVzrqWqrKhN10PMlJlyXf4a+llX69erO7+DwEOAm4HHPatJ2reapAeKfNyJ/FXl8tnCrdwzkWm1TpOs0+U7wDdBDXgR7lut+6QLHEq6wo6jMbXaOMr81G0ObC0rWFnaNaUBX92B5529wT5Y2SfZKZTxf3lHL6/13Hssv+5ivwMu6cE22DQvmBH3TR/68PZPuFZiLT0+tDhkAyG2aStktZu+1+7WoTVQhFsDRa2Bahw6A63Dv3H7GYzoJhVPZgRfCvN1p/xtp/zB+QlCZeUrh+YQ3ZND5FU5hPAwTQCIUwDjCJGENJkA4NAqdGTbUgIwamnuvgTDCRmmGFMaYwwTSJqSj3A6BJhCgNIYRzbJRcdlX/QC7JP2HRz9wBAgEgzf1gYaqlsarmur3do118I6wvWRyBm/B3JaZgyTFCQxSilBNJxMhUwB9rY+lZuIwmEMSYySmCCCE9h8COltPRY1u1noJ5sq1vklH7uzBVtQY3u5uroZhIOE0fm1M7uEcrHZCAt3BJFV2hGocDGUap3Gcl5v0eylsCmLZqbSvDMBbFZimpQvjVa3/EwVynFWKumynrEoipaJFWIi3bzhck3vkctxRMaKU98wE3lePJQyaeVy3rwxK14+SSIBaB91Eg9JJ01CpOdQAx0qS0L7qJKVhxtfVXb3pyZKsuJ8a23Bt73no1JzH8d/uDErf67LKqMeEramqJHHRK1nOfa9Xm1BBntqXo10lyzPVDNCmmsraZ+H7atYFD8y0IFVKWQBr0zNZ9LlcRokh6FBa8NLSLIXDV5sTUleNN1BNG0qQxqht5jw7Bv1V97RNc/WCdiPG90dXdRa0MIW61gS0d3j/w8/hdCodXZF+w6vjvsppDu7b0wl6xzwzQMa0SagNEIdQJOe6Zcc7GtH9wPqk8Xy0MqG9lS2B6A/krJROkxhnERpkkAEWt9ucUyGEBMQOcVL2qPvLXrulCGmIPw1d3IohsOUJu3WZwuirW5/I1Dfvv2lBT7/CQ==</diagram></mxfile>
```

Comment:
I don't think this file needs to go in the repo. Do let me know if I'm wrong about that, though.